context/app/static/js/components/Detail/Dataset/context.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const DatasetContext = React.createContext({});
```

Comment:
Can you add a comment about what can/should be stored in this context?
```suggestion
// Stores the `assetsEndpoint` and `uuid` values.
const DatasetContext = React.createContext({});
```